docs/core/compatibility/aspnet-core/6.0/middleware-ambiguous-https-ports-exception.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "Breaking change: Middleware: HTTPS Redirection Middleware throws exception on ambiguous HTTPS ports"
+description: "Learn about the breaking change in ASP.NET Core 6.0 titled Middleware: HTTPS Redirection Middleware throws exception on ambiguous HTTPS ports"
+author: scottaddie
+ms.author: scaddie
+ms.date: 02/04/2021
+---
+# Middleware: HTTPS Redirection Middleware throws exception on ambiguous HTTPS ports
+
+In ASP.NET Core 6.0, the [HTTPS Redirection Middleware](xref:Microsoft.AspNetCore.Builder.HttpsPolicyBuilderExtensions.UseHttpsRedirection%2A) throws an exception of type <xref:System.InvalidOperationException> when it finds multiple HTTPS ports in the server configuration. The exception's message contains the following text:
+
+```console
+"Cannot determine the https port from IServerAddressesFeature, multiple values were found. Set the desired port explicitly on HttpsRedirectionOptions.HttpsPort."
```

Comment:
I don't normally code fence error messages, but if this is standard for your docs, it's fine.